python/federatedml/util/anonymous_generator_util.py

```diff
@@ -0,0 +1,125 @@
+#
+#  Copyright 2019 The FATE Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+import copy
+import numpy as np
+from federatedml.util.data_format_preprocess import DataFormatPreProcess
+
+
+ANONYMOUS_COLUMN_PREFIX = "x"
+ANONYMOUS_LABEL = "y"
+
+
+class Anonymous(object):
+    def __init__(self, role=None, party_id=None):
+        self._role = role
+        self._party_id = party_id
+
+    @staticmethod
+    def anonymous_migrate(anonymous_header, original_party_id, new_party_id):
+        migrate_anonymous_header = []
+        for column in anonymous_header:
+            role, party_id, suf = column.split("_", 2)
```

Comment:
delimiter which is a global variable may be better

Reply:
> delimiter which is a global variable may be better

It's fixed in this commit in another branch